match.go

```diff
@@ -18,8 +18,8 @@ const separator = "/"
 //	pattern:
 //		{ term }
 //	term:
-//		'*'         matches any sequence of non-Separator characters
-//		'?'         matches any single non-Separator character
+//		'*'         matches any sequence of non-"/" characters
+//		'?'         matches any single non-"/" character
```

Comment:
We can probably match up with https://golang.org/pkg/path/#Match here. (That is no need for quotes.)